client/js/views/templates/OurOfferings.js

```diff
@@ -1,3 +1,16 @@
 module.exports = function( { model } ) {
-    return `<div>our offerings</div>`
+	const offerings = model.reduce((memo, offer) => {
+		const markup = `<div>
+			<h3>${ offer.header }</h3>
+			<div>${ offer.item }</div>
+			<div>${ offer.quantity }</div>
+			<div>${ offer.price }</div>
+		</div>`
+		return memo + markup
+	}, '')
+    return `<div>
+    <h1>Headline for List of Stuff that’s Available</h1>
+    <p>Ar egestas eleifend. Etiam luctus mattis justo, vitae fermentum libero euismod lacinia. Proin at consequat risus. Praesent sollicitudin vestibu.</p>
+    <div>${ offerings }</div>
```

Comment:
Now break this up:
`<div>
    <div><h3>Fall 2017</h3><div>${fallOfferings}</div></div>
    <div><h3>Fall 2017</h3><div>${yearRoundOfferings}</div></div>
</div>`